tests/Panel/FileTest.php

```diff
@@ -21,14 +23,101 @@ public function isLocked(): bool
  */
 class FileTest extends TestCase
 {
+    protected $app;
+    protected $fixtures;
+
+    public function setUp(): void
+    {
+        $this->app = new App([
+            'roots' => [
+                'index' => $this->fixtures = __DIR__ . '/fixtures/FileTest',
```

Comment:
Temporary files in test shouldn't be stored in `/fixtures`, but in `/tmp`. This allows us to easily differentiate between files that should be in the repo (= fixtures) and files that shouldn't (= tmp).

Reply:
✅